doc/source/ray-air/examples/training_ingest_object_classification.ipynb

```diff
@@ -0,0 +1,1232 @@
Kx23mfm5caSJb6z07JZN9+JpPh/kzO3z73d/Pr+/fvH6zbO350s6XBZ1B+/Nf//Ff/dlffnpXP3z39z989beXx99qnC9PW6Skl7ZhWbsniJI4pXuEtOZJgZbMCIRWEZNap140Mlvz1tpUT4BFBJ0EVGsSw82yFOueZiNeOwGqFjEFpPeGUiSDdGb0HuKhJhJlnufDVMd0PIp1KSWarC2Tdl6Xo83Pa0vR1SG9tubBGqJLy54bVUgVWPf0TNF5PE9GRHiGMYugDTqMkEiXnZw7INcRDZ5aBClj/4edA7x/+KI0s1J0sPmsKt2GgTV2I5cE0HvPQGSQMY9AXwY4XF/G7ZPMtZMmcCokALUiApMYTruDrzySmIaiWq44ww5WAmBmOkEOGmYpKrvMDowsAq1TONYVm1udjlofFJNcTU0CHqkoiBBLj0APKWFmUymlKqrsfCrVVCtCFbKalFImsy6qTC21Ho7TdDKYpF4mNSk6H+5effrw81/bZ1/Km8+Xw/GgvW+pAaYq7HA4aS3n5dmkujvTE1pUSQma1RrdTS2ji07INKtMV9UMiXGEYqSvc4BT0TOZ4+JRcAwN5KhWclP7kgxP0E2Fyaenx8wv2+beU6SYzgPaNwEstMrdq+OHD2enV61UG0gRKdcGTneLWQiGBc/VE6EghtTlR91KAIPdNEQAP6oOkYNYz1Fkr1XnZe24VZAftWwQedET7ftl+biAfPlvf/L1k6d9+VeDf/jykb3q+5Xkdo0NGnvJnX0jQubLyIxw9t57dmh2354u53VZwjPPz5+8mueJ5+f3oUeVmpD5cPjXf/LXf/0Xf/7pw/H9N3/7/tt//PDd77blQ9/WiLl3by7dMygRnqCYzpNdlq3H1pNQyejJqNXAKkxhquwbj8wMhrGUohGsU8kgYB5ZywRNSGTm2CNvHp1MNmHPLCKCyH0DjDCz8O4JEZu9RnayEBCRD21rwLquWqbnSzNObds0y9bx+LhaPUQPEVu7d49aq4DuiBi+DDKyaTwCqCRzzDUyYj5EFdFFTUbaUca1GSc4gp4wPItuLD+12qAxnORUh+BZSSC3ZE8EOALiTVlFC+Akhy0adPTOSdLbU2ZCa8KAITydQNpY78i1EZdUGeTxaX9YbLCxwjMzQ3Nkr+sOxu4OodkXLQXCZe3P5+5RpuNrKw+DRhc+zKcAmmhVA3lOMnqyTKzF9DiZTCppEtEVMfwuFFIqpsk0+1xNtJpZsVpt0nJUmfoxjvef1rvP59e/qq8+3+ohSnGn1dOhTpPU6C0iREs9zJv3WBfRhMqhTOVwiKB1Mz0epjkpl8tl1a1FQJOJhEQg8xpCw5BduqRxzWEFbkMSMrNaJYcHxX6bRYS7m0WxKXP4MglggGSYGYdfoKhUw+FYf/hhPUxVQq/Mh13gtZuXoqgqqbtk+spSGiE/lD11+eNoNsIBCQ3d6TDjQ03vebUf3J0SsHsgALipGG9lRUc1Gf6014LClETcHrgWo9EH7UjZS069yNVIQgQvGKEA4mP5GtHn16e7LqtvpXP/6V5WtyvVCwRCfI9rRqZH67s1G7qJA1nqcdJyhv3ii1/8qz//sz//5Z/SLz/8/u+++/1/RD5eLt8/PV3CbQTbiqUxsy9bXyKzlPK8Ru/RuamZ1UKQvRcVpQxOCkVrnY5SEpKdAjMtgx+cDAGBgLD3zd0GJb6qUeFMIhAro5Ij/m7n9ZVink5ZVfVwmJw+Zqfeu0u0DM9YP2wJG3ngmUmWbetSzH2b58ocS/TCa/p20pFXFgKgmgolwGTsPOkU0SF2H3QjDg45h8+3jWtvZKOM6Q1AMZCuQtEiSlOkDn2bjx0CmSY1kGXYGWcDhMgMKBAYedxRsfvpkaEqzJ0StDsT79o2jvbKRHOk2smIG5fb1SE7V1uGzbJQRbRQUmBWPGRZtq2nlLnUU7ETSI+1tcV9GwZhQqgWjUSCI22UiqSUIqivTodkSwYyhsNULYDBRGzSMhVTFIkys9zP5fhwfPhZefh8PX7SD5+2coqIKXGgb6i9Z4ttsjIf7tWXFKYns2U6c+0iEvNO/FGd6ynBCB762tsS2d17eEQb3s8pu9hwUMVY52m0t3tjdeXyuvvgwApRVKDmGUm6+8CCnp6erhKFKiJq0rMhQ1TncsjN4TEV88w9FhchARFKilJbG/fxj8ywAJTx7MPm73rUCCSVZacQ4HoNkgB1kI2Ze9YOx5KARQs5rszgDm/zuuvbx7xhHf2TFmmvz8TotpTBvS/a50Neu6QffV29518kS9+MtDDAr50VwT1Ie7zWVUC0S4UQg8Caw7ozkNmjNRcxs5ou5e6TZVtmMxO7PK9/9PNf/tGXb0rm26/+Q18uT++/f373dcS5+9Z6XBY/3h+KGSN69oht25YMxSTn5661FJugnKaiECQtS622bE2pHgLTUsrWV9FEVwrNrGdYNfFuKhTPBCPGliCrZOoYrsQtRSKyRxhspMUlCVOH6FSm4zRNtVYD+7YtkpLeSD49PU2Hu8vS6+GubT2SIiNnau9qj9OBUGqIqqhEjF3MuAEgbGoqqiIHku7NVFUoNrJAx8pmcAlAoKrs14cA18hfABLCTEiaKMPJLJpAKi2SpWggzJDeVIuBWuuQPYeEkMJEBtCLzYmsZSZpZp6hFoEo+wbZKIP9vS+y90NXbKiWh/WwqJZiqiqGwXYc1msgVCtQ1mVbLg1ArVUVotGX6G1t69r9LIpShhUnjENwSEEffW96hERb0kqaZSIxOhowmfN0kMlK0cmmOt9Pr760T7/E3Wu7+3XWuac0CugqgGYkZYtjOWQtZERwaRtAFWSkCbeIHit0LnbIZGS67LhVKUVMkRgS5VvfMAwqrlxGHSareeVzD5IHkp4uArOhoqQpxIxqTPUEEe/evSdJBsTVxH34aJnp/ObV5+/evbMsQrgYx89+nRr2XWS54VmqRbSM9SBL3uIhhnpqfKkWKQQgORJ0doIJ8lB3DCt32z8dP+suIYu
```

Comment:
Nit: Here and elsewhere, should we use a Jupyter output instead of illustrating the output in a comment? Might be easier to read, and would avoid duplication 
![image](https://github.com/ray-project/ray/assets/26107013/fd109e19-0e0c-4efd-9a47-1a025f13fb58)